build/Makefile

```diff
@@ -212,6 +220,9 @@ ifeq ($(PF_ARCH),x64)
 	cd $(OMI_ROOT); $(RMDIR) output; ln -s output_openssl_1.1.0 output
 	$(MAKE) -C $(DSC_DIR) dsc110
 	$(MAKE) -C $(DSC_DIR) dsckit110
+	cd $(OMI_ROOT); $(RMDIR) output; ln -s output_openssl_3.0.0 output
```

Comment:
this is already done 4 lines above except the linking part

Reply:
ah, it is from merge probably, will take a look. thx

Reply:
I think it needs to stay to make sure CMAKE does not change the current folder, also to clean up inter output dir before building 300 kits. same is written for 100 and 110 paths.